app/assets/stylesheets/notifications.scss

```
@@ -415,6 +417,8 @@
   }
 
   .load-more-wrapper {
+    z-index: 0;
```

```suggestion
    clear: both;
```

...but about other elements on the page using `float: left;` properties to set the layout. It does weird thing to elements after that, including `.load-more-wrapper`. Adding `clear: both;` should do the trick.